.github/workflows/fuzz.yml

```diff
@@ -49,7 +49,7 @@ hashes_sha1,
           key: cache-${{ matrix.target }}-${{ hashFiles('**/Cargo.toml','**/Cargo.lock') }}
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.58
+          toolchain: 1.65.0
```

Comment:
Yaml is complicated, 1.60 will be rounded to 1.6.
As a best practice use strings.
Instead of `1.60` use `'1.60'`.
Also check [YAML Hell](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)

Reply:
I am future-proofing this in #2112.

Reply:
yeah, or just add the full version. This is a matter of preference 😄 but the problem happens also if we have `1.65.10` so like to have a safe guard, thanks